packages/rehype-mathjax/lib/output-chtml.js

```diff
@@ -0,0 +1,7 @@
+const CHtml = require('mathjax-full/js/output/chtml').CHTML
+
+module.exports = createOutput
+
+function createOutput(options) {
```

Comment:
According to README, should we add validation? For instance,

```js
if (!options.fontURL} {
   console.error(`options.fontURL is '${options.fontURL}'. You must set this option with URL to reach MathJaX fonts.`)
}
```

Reply:
Oh that’s a good idea. CHTML without `fontURL` doesn’t make sense at all, on a server, right?

If that’s the case, then I’m thinking of `throw new Error` instead of `console.error`, to really crash. What do you think?

Reply:
Yes. At least, on server side rendering, we should set this parameter because the stylesheet refers to `/MathJax_*.woff` if `fontURL` is undefined. That is, It no problem if users put fonts this path correctly.